int32[] blob_id
geometry_msgs/Point[] center_position
geometry_msgs/Point[] center_velocity
geometry_msgs/Point[] top_position
geometry_msgs/Point[] top_velocity